gnomad_qc/v4/variant_qc/random_forest.py

```diff
@@ -0,0 +1,353 @@
+# noqa: D100
+
+import argparse
+import json
+import logging
+import sys
+import uuid
+from typing import List, Optional, Union
+
+import hail as hl
+from gnomad.resources.grch38.reference_data import (
+    telomeres_and_centromeres,
+)
+from gnomad.utils.slack import slack_notifications
+from gnomad.variant_qc.pipeline import train_rf_model
+from gnomad.variant_qc.random_forest import (
+    apply_rf_model,
+    get_rf_runs,
+    get_run_data,
+    load_model,
+    median_impute_features,
+    pretty_print_runs,
+    save_model,
+)
+
+from gnomad_qc.slack_creds import slack_token
+from gnomad_qc.v4.resources.annotations import (
+    get_vqsr_filters,
+    get_variant_qc_annotations # since get_info and others are being rolled into one
+)
+
+from gnomad_qc.v4.resources.basics import get_checkpoint_path
+from gnomad_qc.v4.resources.variant_qc import (
+    get_rf_annotations,
+    get_rf_model_path,
+    get_rf_result,
+    get_rf_training,
+    rf_run_path,
+)
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("variant_qc_random_forest")
+logger.setLevel(logging.INFO)
+
+FEATURES = [
```

Comment:
As discussed, think of a logical place to move this and also be used by the annotations script. Also the other constants might be better moved to another location too